csrc/ir/internal_nodes.h

```diff
@@ -2255,4 +2255,37 @@ class NVF_API CatOp : public Expr {
   Val* getPred(int input_idx) const;
 };
 
+//! Linear Operator to be expression evaluated without decomposition.
+class NVF_API MatmulOp : public Expr {
+ public:
+  using Expr::Expr;
+
+  MatmulOp(IrBuilderPasskey, Val* out, Val* in_a, Val* in_b);
+
+  NVFUSER_DECLARE_CLONE_AND_CREATE
+
+  const char* getOpString() const override {
+    return "LinearOp";
```

Comment:
Is this a `F.linear` op or a `torch.matmul` op? This line should match the class name and I think we should have separate ops and nodes for each of `linear` and `matmul`. This PR should only introduce one of these ops so we can inspect the mapping rules one op at a time.